x/nft/go.mod

```diff
@@ -156,3 +156,4 @@ require (
 // Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
 // TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+replace github.com/cosmos/cosmos-sdk => ../../
```

Comment:
Also to be removed

Reply:
ditto: https://github.com/cosmos/cosmos-sdk/pull/15520#discussion_r1147767292